packages/@ourworldindata/grapher/src/core/GrapherAnalytics.ts

```diff
@@ -2,30 +2,30 @@ import { findDOMParent } from "@ourworldindata/utils"
 
 const DEBUG = false
 
-// TypeScript implicitly imports @types/amplitude-js and @types/google.analytics
-// so we have proper types for window.amplitude and window.ga
-
-// Docs on GA's event interface: https://developers.google.com/analytics/devguides/collection/analyticsjs/events
-interface GAEvent {
-    hitType: string
-    eventCategory: string
-    eventAction: string
-    eventLabel?: string
-    eventValue?: number
+// Add type information for dataLayer global provided by Google Tag Manager
+type WindowWithDataLayer = Window & {
+    dataLayer?: GAEvent[]
 }
-
-enum Categories {
-    GrapherError = "GrapherErrors",
-    GrapherUsage = "GrapherUsage",
-    GlobalEntitySelectorUsage = "GlobalEntitySelector",
-    SiteClick = "SiteClick",
-    KeyboardShortcut = "KeyboardShortcut",
+declare const window: WindowWithDataLayer
+
+export enum EventCategory {
+    CountryProfileSearch = "owid.country_profile_search",
+    Filter = "owid.filter",
+    GlobalEntitySelectorUsage = "owid.global_entity_selector_usage",
+    GrapherClick = "owid.grapher_click",
+    GrapherError = "owid.grapher_error",
+    GrapherUsage = "owid.grapher_usage",
```

Comment:
Can the `GrapherUsage` event category be removed now that it is no longer used in `logGrapherTimelinePlay`? Not sure if it is referenced elsewhere beyond this file.

Reply:
Good call!